### PR TITLE
A bit material design

### PR DESCRIPTION
**Pull request by larsgrefer**

### CODE REVIEW

**Comment by SecUpwN** [comment]
Huge thanks for this! The update on the Drawer Toggle is causing Travis CI to fail, please see the [logfile](https://travis-ci.org/SecUpwN/Android-IMSI-Catcher-Detector/builds/94756241).

**Comment by larsgrefer** [comment]
@SecUpwN There seems to be an error with the AVD on travis. I'll have a look at it later

**Comment by SecUpwN** [comment]
>There seems to be an error with the AVD on travis. I'll have a look at it later

Thanks, I'll be wating with merging until a fix has been added here.

>If we wan't to inform about a status change, we should use a "peeking notification":
>https://www.google.com/design/spec/patterns/notifications.html#notifications-peeking-notifications

Since we likely cannot circumvent that our app gets displayed incorrectly some day when Google decides to changes more stuff and guidelines of the AOS (don't ask how much we hate that), we've been re-thinking the Icons in the navigation drawer and would be happy if you'd chime in on #512 to solve this.

**Comment by agilob** [comment]
@SecUpwN you linked to peeking notification, but app uses ongoing notification now. Is anyone planning removing ongoing notification or am I missing something?

File: app/build.gradle [hunk 1]
```diff
@@ -99,11 +99,10 @@ android {
 dependencies {
     // DO NOT REMOVE BELOW COMMENTED-OUT CODE BEFORE ASKING!
     //compile 'com.github.amlcurran.showcaseview:library:5.0.0'
-    //compile 'com.android.support:appcompat-v7:22.1.1'
+    compile 'com.android.support:appcompat-v7:22.+'
```
Comment (agilob) [inline app/build.gradle]: this can cause unexpected behaviour depending on build machine. Please specify version.

**Comment by larsgrefer** [comment]
@agilob @SecUpwN I think there should be one persistent notification with low priority (in order to keep the service alive) and an additional peeking notification with high priority on status changes.

I like this Idea: https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues/512#issuecomment-125046784

File: app/build.gradle (hunk 1 above)
Comment (larsgrefer) [inline app/build.gradle]: :+1:

Comment (agilob) [inline app/build.gradle]: I found it out really hard way...

**Comment by SecUpwN** [comment]
@larsgrefer, the idea from https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues/512#issuecomment-125046784 is not final yet, let's discuss the usage of notifications there. Current priority on this PR is to solve the following things (thanks ahead for digging deeper):

* Will Issue #479 reappear if we increase the `targetSdkVersion`?
* Version of `appcompat` needs to be specified as per https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/pull/654#discussion_r46675610
* Failing Travis CI build needs to be resolved before I merge this (see [logfile](https://travis-ci.org/SecUpwN/Android-IMSI-Catcher-Detector/builds/94756241)).

**Comment by larsgrefer** [comment]
@SecUpwN #479 was already fixed in #527, was'nt it?

**Comment by larsgrefer** [comment]
@SecUpwN what is tested by `testGetPropsReturnsValue()`?
This one fails

**Comment by agilob** [comment]
LOL
Feel free to remove this "test" it tries to start an actual activity on Android, it fails because there is no device with screen on attached to gradle test.

**Comment by SecUpwN** [comment]
>@SecUpwN #479 was already fixed in #527, was'nt it?

@larsgrefer, partially. The main "fix" to show the colored notification was https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/commit/deb50bfc7125f601707162a2b4c3f206e52af5bb, which will likely be removed when you update the `targetSdkVersion` from `19` to `22`. Hence my previous question.

>Feel free to remove this "test" (whole file) it tries to start an actual activity on Android, it fails because there is no device with screen on attached to gradle test.

@agilob, don't we need that `testGetPropsReturnsValue()` in the future for something like #173?

**Comment by larsgrefer** [comment]
#656 should fix the travis errors

**Comment by SecUpwN** [comment]
Thanks, that indeed fixed the Travis CI check. @larsgrefer, last question regarding this PR: Will #479 come back when we merge this? The purpose was to keep the notification icon *colored*, I expect that with changing the `targetSdkVersion` we will likely have a white Icon up there which does not change color accordingly, which in turn makes our [Status Icons](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/wiki/Status-Icons) pretty much useless. Thanks for your endurance on this Issue.

**Comment by larsgrefer** [comment]
If we merge this the statusbar-icon will be just a white shape like here: https://github.com/larsgrefer/Android-IMSI-Catcher-Detector/commit/5ac27631a6a8a5b2e71d34c7a8206c5b71d78cc2#commitcomment-14785284

But the icon of the full notification will be colored like here: https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues/479#issuecomment-118618175

I don't think we should stay on pre-lollipop and block material design just to keep a colored status bar icon

In order to inform about a status change i'd suggest to use a "peeking notification":
https://www.google.com/design/spec/patterns/notifications.html#notifications-peeking-notifications

**Comment by SecUpwN** [comment]
>I don't think we should stay on pre-lollipop and block material design just to keep a colored status bar icon

Makes sense, but the reason I previously opened #479 was because users would always have to pull down the notification bar in order to see the current state of our our app (as long as we have not implemented peeking notifications). @larsgrefer, please add the link and your thaoughts about peeking notifications to #512. Regarding the merge of this PR, I would like to hear the opinion of @He3556 as well.

**Comment by larsgrefer** [comment]
@SecUpwN Good news: with `compileSdk` 22, `appcompat` 22 and `targetSdk` 19 we can have colored statusbar-icons AND material design, so #479 won't come back

**Comment by larsgrefer** [comment]
In fact there is a running avd in an emulator
Am 04.12.2015 10:50 nachm. schrieb agilob <notifications@github.com>:LOL
Feel free to remove this "test" it tries to start an actual activity on Android, it fails because there is no device with screen on attached to gradle test.

**Pull request merged**